libr/include/r_cons.h

```diff
@@ -487,6 +493,8 @@ typedef struct r_cons_t {
 
 #define R_CONS_KEY_ESC 0x1b
 
+#define ANSI_ESC_CLEAR_LINE "\r\x1b[0K\r"
```

Comment:
its the only key prefixed with ANSI_. we may find a better name. the first \r is probably not needed. i think it was added to fix some bad behaviour in some terminal emulators.